Makefile

```diff
@@ -103,6 +107,20 @@ woke: woke-install
 pa11y: pa11y-install html
 	find $(BUILDDIR) -name *.html -print0 | xargs -n 1 -0 $(PA11Y)
 
+vale-get: install
+	. $(VENV); python3 $(SPHINXDIR)/get_vale_conf.py
+	vale > /dev/null 
+
+vale-run: vale-get
```

Comment:
Not sure I'm a fan of always retrieving the styles again, even if they haven't changed - but we can see how this works and if anyone complains.

Reply:
> and if it does, it doesn't pull the files

Really? It doesn't create the directory if it already exists - but I don't see it stopping anywhere before pulling the files.
I'm actually not sure if it should or shouldn't update them every time. I just want to make sure whatever we do is a conscious decision (which it doesn't seem to be :wink: ).